### ISSUE TITLE
cdat_info.py exists only as text inside its conda receipt

### ISSUE THREAD

**Issue by danlipsa** [UV-CDAT/uvcdat#2047]
cdat_info.py does not exist as a proper python file. It exists only as text inside its conda's build.sh.
This makes it difficult to edit the file - this file leaks some resources that need to be fixed.

**Comment by danlipsa** [comment]
@doutriaux1 @aashish24 Why is this? Can we move it in a proper repository?

**Comment by doutriaux1** [comment]
@danlipsa yes we should do this. I will take care of this. It's mostly a legacy issue, it used to be part of the install.py.in and was already created manually since the 90s.... Time to upgrade.
